itest/rust/src/register_tests/keyword_parameters_test.rs

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+// Needed for Clippy to accept #[cfg(all())]
+#![allow(clippy::non_minimal_cfg)]
+
+use godot::classes::{EditorExportPlugin, IEditorExportPlugin};
+use godot::prelude::*;
+
+#[derive(GodotClass)]
+#[class(base=EditorExportPlugin, init)]
+struct KeywordParameterEditorExportPlugin {
+    _base: Base<EditorExportPlugin>,
+}
+
+#[godot_api]
+impl IEditorExportPlugin for KeywordParameterEditorExportPlugin {
+    // This test requires that the second non-self parameter on `export_file`
+    // remain named `_type`. Additionally tell rustfmt to skip this code and
+    // allow all clippy lints.
+    #[allow(clippy::all)]
+    #[rustfmt::skip]
+    fn export_file(&mut self, _path: GString, _type: GString, _features: PackedStringArray) { }
```

Comment:
Why rustfmt and clippy? Are they really needed? If yes, can you limit clippy to specific lints?

Reply:
The idea there is to protect the `_type` parameter from getting renamed by some random future change to clippy or rustfmt and then missed since that wouldn't make the test fail. 
It's quite an over the top move considering there's no reason to expect that to happen. I can just remove it of course

Reply:
Yes, please let's avoid defensive programming unless we know there are actual issues. 

Rustfmt renaming identifiers would be absolutely insane and break a ton of generated code, this is exceedingly unlikely to happen. If clippy ever becomes unhappy, we can address that specifically, as well 🙂